<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>conquest_ui::NewSandboxDialog</class>
 <widget class="QDialog" name="conquest_ui::NewSandboxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New sandbox window</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="blue_player_type">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>AI player</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Human player</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Handcrafted</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Random orders</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Red:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="red_player_type">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>AI player</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Human player</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Handcrafted</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Random orders</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="blue_player_generation">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="prefix">
    <string notr="true">Gen </string>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="red_player_generation">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="prefix">
    <string notr="true">Gen </string>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>blue_player_type</tabstop>
  <tabstop>blue_player_generation</tabstop>
  <tabstop>red_player_type</tabstop>
  <tabstop>red_player_generation</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>conquest_ui::NewSandboxDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>conquest_ui::NewSandboxDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
